Applications/Games/Uplay/Online/script.js

```diff
@@ -1,17 +1,24 @@
 include("engines.wine.quick_script.online_installer_script");
+include("engines.wine.plugins.windows_version");
+include("engines.wine.verbs.corefonts");
 
 var installerImplementation = {
     run: function () {
         new OnlineInstallerScript()
             .name("Uplay")
             .editor("Ubisoft")
             .applicationHomepage("https://uplay.ubi.com/")
-            .author("Plata")
+            .author("Plata, KREYREN")
             .url("https://ubistatic3-a.akamaihd.net/orbit/launcher_installer/UplayInstaller.exe")
             .category("Games")
             .executable("UbisoftGameLauncher.exe")
             .wineVersion(LATEST_STAGING_VERSION)
             .wineDistribution("staging")
+            .preInstall(function (wine/*, wizard*/) {
+                wine.corefonts();
+                wine.setOsForApplication().set("upc.exe", "winvista").do();
```

Comment:
Why not win7 ? ^^

Reply:
Because wineHQ stated vista and i'm lazy to test all options

Reply:
Ok then.

Reply:
Also because win7 has rendering issues as referenced above